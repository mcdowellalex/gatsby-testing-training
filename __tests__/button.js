// testing css in js

import React from "react"
import styled from "@emotion/styled" //docs had react-emotion but this is the new package
import renderer from "react-test-renderer"

const Button = styled.div`
  color: hotpink;
`

test("Button renders correctly", () => {
  expect(
    renderer.create(<Button>This is hotpink.</Button>).toJSON()
  ).toMatchSnapshot()
})