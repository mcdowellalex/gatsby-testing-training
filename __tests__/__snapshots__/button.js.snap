// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
.emotion-0 {
  color: hotpink;
}

<div
  className="emotion-0 emotion-1"
>
  This is hotpink.
</div>
`;
